tests/conftest.py

```diff
@@ -80,3 +85,26 @@ def create_authorized_user(postgres):
 
     with step("Removing user from DB"):
         postgres.delete_user(user_to_create["id"])
+
+
+@fixture(scope="function")
+def creating_user_via_api():
```

Comment:
Maybe `create_and_delete_user_via_api` and and instead of `return` yield + steps for delete user?

Reply:
Fixed